components/mime.rst

```diff
@@ -0,0 +1,577 @@
+.. index::
+   single: MIME
+   single: MIME Messages
+   single: Components; MIME
+
+The Mime Component
+==================
+
+    The MIME component allows manipulating the MIME messages used to send emails.
```

Comment:
We also have the MIME type guessers, which should probably be documented here somewhere.

Reply:
Yes! I've added the changes made in #10886, so I'm closing that PR.